exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,18 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var percentSign = "%"
+var total = numberOfStudents + numberOfMentors;
+/*calculation for students*/
+
+var text1 = "Percentage of Students ";
```

Comment:
Great understanding of basic javascript syntax and naming conventions. No need to understand all the words, as long as you understand the concepts.